src/debug/daccess/daccess.cpp

```diff
@@ -4489,6 +4490,44 @@ ClrDataAccess::TranslateExceptionRecordToNotification(
             break;
         }
 
+        case DACNotify::JIT_NOTIFICATION2:
```

Comment:
This code also is almost exactly like JIT_NOTIFICATION maybe there is a way not to duplicate.

Reply:
We can get rid of JIT_NOTIFICATION. The interface between coreclr and DAC is tightly bound so we don't need to preserve back-compat there. Back-compat is useful between tool <-> DAC (The IXCLRData* interfaces) because sometimes we use old SOS with a new runtime/DAC. Tools like windbg and CLRMD also don't version tightly.